### PR TITLE
Implement/loginpage

### PR DESCRIPTION
**Pull request by ArdyZ**

### CODE REVIEW

File: components/TextIamgeTransparent.vue [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<template>
+    <div class="textimagetransparent">
+        <img class="image" :src="image" />
```
Comment (dsluijk) [inline components/TextIamgeTransparent.vue:3]: `img` tags should be `NuxtImg` tags, like `<NuxtImg src="/images/gamers.jpg" />`

File: components/TextIamgeTransparent.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    margin: 0.5rem 0rem;
+}
+
+.textimagetransparent {
+    position: relative;
+    text-align: left;
+    color: white;
+}
+
+.link {
```
Comment (dsluijk) [inline components/TextIamgeTransparent.vue:58]: CSS selectors should just use the component name, this makes the classes redundant

File: pages/login.vue [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<template>
+    <div class="login">
+        <TextIamgeTransparent image="/images/gamers.jpg"></TextIamgeTransparent>
```
Comment (yoshivda) [inline pages/login.vue:3]: I suggest renaming this component to `TextImageTransparent`, so without the typo

**Review by dsluijk — CHANGES_REQUESTED** [review]

**Review by yoshivda — CHANGES_REQUESTED** [review]
File/component needs renaming to get rid of the typo

**Review by dsluijk — APPROVED** [review]

**Review by yoshivda — APPROVED** [review]

**Pull request merged**